.github/workflows/build-release.yml

```diff
@@ -2,24 +2,61 @@ name: Build
 on: [push, pull_request]
 
 jobs:
-    deploy:
-        name: build dependencies & create artifact
-        runs-on: ubuntu-latest
-        steps:
-            - name: Checkout
-              uses: actions/checkout@v2.0.0
-
-            # Install PHP dependencies (Production ONLY)
-            - name: Install composer dependencies
-              run: composer install --no-dev -o
-
-            # Remove development files
-            - name: Clean-up project
-              uses: PrestaShopCorp/github-action-clean-before-deploy@v1.0
-
-            # Zip files and upload to artifacts list
-            - name: Create & upload artifact
-              uses: actions/upload-artifact@v1
-              with:
-                  name: ${{ github.event.repository.name }}
-                  path: ../
+  deploy:
+    name: build dependencies & create artifact
```

Comment:
I guess it's too late to use https://github.com/PrestaShop/.github/pull/2 instead?

Reply:
Never too late.

Reply:
@atomiix  #50 for you : :heart: